__tests__/app.test.js

```diff
@@ -0,0 +1,114 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const res = require("express/lib/response");
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of objects of all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body).toBe("object");
+      });
+  });
+  test("200: returns topics as correct format", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        res.body.results.forEach((topic) => {
```

Comment:
Not technically wrong, but you could include this within the then block of the test above to keep your code dry-er.